src/entities/mellow/config/utils.ts

```diff
@@ -14,7 +13,7 @@ export const disableMaturedWeights = (config: NetworkConfiguration): NetworkConf
             return {
               ...vault,
               weight:
-                Date.now().valueOf() > vault.maturityTimestampMS - 48 * ONE_HOUR_IN_MS
+                Date.now().valueOf() + 48 * ONE_HOUR_IN_MS > vault.maturityTimestampMS
```

Comment:
perhaps this 48h should be a constant of its own. I would prefer to change stuff in constants.ts if tomorrow we say 48 should be 72. WDYT?